test/utils/rpc-errors.test.js

```diff
@@ -0,0 +1,17 @@
+const nearlib = require('../../lib/index');
+const { parseRpcError, AccountAlreadyExists, TxExecutionError } = nearlib.utils.rpc_errors;
+
+test('test parseRpcError', async () => {
+    let rpc_error = {
+        TxExecutionError: {
+            Action: {
+                index: 1,
```

Comment:
what if there are errors in multiple actions? Generally I think test should check more examples ideally as parsing code is non-trivial.

Reply:
> what if there are errors in multiple actions?

That's interesting.. Right now we consider 1 result = 1 error. Maybe we would want to change it in the future, but it would require quite a bit of changes of how node is works.

> Generally I think test should check more examples ideally as parsing code is non-trivial.

Yeah, sure

Reply:
@fckt so is it returning error on the first failed action? Does it even try to execute other ones or stops there? If it stops there, I think it's fine, it is simple enough model.

Reply:
> Does it even try to execute other ones or stops there?

yeah, it fails on the first error